modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/operationPolicy/OperationPolicyTestCase.java

```diff
@@ -68,12 +98,26 @@ public class OperationPolicyTestCase extends APIManagerLifecycleBaseTest {
     private final String API_CONTEXT = "AddNewPolicyAndInvokeAPI";
     private final String API_END_POINT_POSTFIX_URL = "xmlapi";
 
+    private final String TEST_POLICY_NAME = "customCommonLogPolicy";
+    private final String TEST_INVALID_POLICY_NAME = "customCommonLogPolicyInvalid";
+
+    private final String TEST_POLICY_VERSION = "v1";
+
+    private final int TEST_POLICY_LIMIT = 100;
+    private final char[] PUBLISHER_USER_PASS = "pass@123".toCharArray();
+    private String publisherUser = "importExportPublisher";
+
     private String applicationId;
     private String apiId;
     private String newVersionAPIId;
     private String accessToken;
     private Map<String, String> policyMap;
 
+    private File exportedOperationPolicyZip;
+    private String exportUrl;
+    private RestAPIPublisherImpl restAPIPublisherExport;
+//    private String publisherURLHttps;
```

Comment:
Do we need this?

Reply:
Fixed by [669abe4](https://github.com/wso2/product-apim/pull/12938/commits/669abe4beafa46b92f046a94eb15766805cd74fa)

Reply:
Fixed by [a7ae252](https://github.com/wso2/product-apim/pull/12938/commits/a7ae2523e0b98b3d6c24c4c351b07dae9290fa3d)